src/app/docs/autonumeric/autonumeric-docs.component.html

```diff
@@ -4,22 +4,32 @@
   pageTitle="Autonumeric"
 >
   <sky-docs-demo-page-summary>
-    The autonumeric module automatically formats currency and other numbers that users enter in form inputs.
+    The autonumeric directive automatically formats currency and other numbers that users enter in form inputs. This directive wraps the third-party autoNumeric library. If you have never used autoNumeric, <a href="https://github.com/autoNumeric/autoNumeric#what-is-autonumeric">the "What is autoNumeric?"</a> guide covers the basics. For more in-depth information about autoNumeric and its capabilities, see <a href="https://github.com/autoNumeric/autoNumeric/blob/next/README.md">the autoNumeric documentation.</a>
   </sky-docs-demo-page-summary>
 
-<!--
-  <sky-docs-demo>
-    See documentation for how to setup demos
-  </sky-docs-demo>
--->
-<!--
   <sky-docs-code-examples
     [packageDependencies]="{
-      '@skyux/autonumeric': '*'
+      '@skyux/autonumeric': '*',
+      'autonumeric': '*'
     }"
   >
-    See documentation for how to setup code examples.
+    <sky-docs-code-example
+      heading="Basic"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/autonumeric/basic/"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Currency"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/autonumeric/currency/"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Presets"
```

Comment:
```suggestion
      heading="Predefined options"
```
I don't think "presets" actually means anything. And looking at the autoNumeric docs, it's not a term we are copying from there. I _think_ "Predefined options" is what we actually mean here. And I think we should remove "preset" elsewhere in the docs. I'll add a separate suggestion for that.